README.md

```diff
@@ -144,25 +144,32 @@ Contributions are welcome! Please feel free to submit a pull request.
 When developing, use `pip install -e '.[dev]'` to install dev dependencies such
 as linter and formatter.
 
-How to run tests:
+- How to run tests:
 
 ```sh
 pytest
 ```
 
-How to run some of the tests, and re-run them whenever you change a file:
+- How to run some of the tests, and re-run them whenever you change a file:
 
 ```sh
 tp -i test ... # replace with path to tests/directories
 ```
 
-How to format:
+- How to run locally without XPK:
```

Comment:
I think a clear description can be "How to run locally inside a docker container"

We can also point out a benefit of this approach, which is "When running inside the docker container, it will use the same dependencies and build process as used in the XPK approach, improving the hermeticity and reliability."